src/ng/location.js

```diff
@@ -1,4 +1,5 @@
 'use strict';
+/* global stripHash: true */
```

Comment:
`stripHash` is used more in `$location` than `$browser` and other similar helpers (e.g. `stripBaseUrl`, `stripFile`) are also in `$location`. I only moved `trimEmptyHash`, because it is only used in `$browser` now.

Even if we had imports/exports (which we don't because everything lives inside the great AngularJS IIFE :grin:), I don't see two way importing going on. `$location` would export `stripHash` and `$browser` would import `stripHash` and export `getHash` and `trimEmptyHash`.

(We definitely have enough URL-specific helpers that we could move them into a separate file and "import" from there :grin:)

Reply:
I think it's a little weird having `$browser` depend on `$location`. Or was that always the case?